docs/operations/disaster_recovery_garden.md

```diff
@@ -0,0 +1,207 @@
+    * `garden.spec.virtualCluster.gardener.gardenerAPIServer.encryptionConfig.resources`
+
+### Step 8: Start Restoration
+
+**Scale Up `gardener-operator`:** Scale the `gardener-operator` deployment back up. It will now reconcile the `garden` resource with the correct initial status.
+
+### Step 9: Restoring multi-member ETCD clusters (HA configuration)
+
+* For a Garden running with an **HA configuration**, restoring **the etcd cluster requires manual steps**.
+* Follow the specific procedure documented in the `etcd-druid` repository:
+    * **Reference:** `https://github.com/gardener/etcd-druid/blob/master/docs/usage/recovering-etcd-clusters.md`
+* Apply the steps to both `etcd-main` and `etcd-events` clusters.
+
+-----
+
+## Edge Cases and Special Considerations
+
+#### 1\. Restoring from Credentials Rotation Phase `Prepared`
+
+When restoring from the `Prepared` phase, the secrets deployed in Step 1 will have two suffixes: the content hash and a second suffix indicating a non-empty `last-rotation-initiation-time` label.
+
+#### 2\. Restoring from Credentials Rotation Phase `Preparing`
+
+If the cluster is being restored from the `Preparing` phase (meaning rotation was actively in progress), you must verify the content of the secrets and the encryption state in **etcd** to ensure consistency.
+
+#### 3\. Encryption Keys
+
+When performing a restore, ensure that the encryption keys and configurations deployed match the state that existed in the former runtime cluster.
+Mismatched keys or an incorrect rotation status can cause the `gardener-operator` to issue new encryption keys. This in turn will render existing data inaccessible.
+Depending on the conditions it might also cause data in etcd to be encrypted with new keys, leading to permanent data loss. Having a separate backup of the etcd backups mitigates this risk.
+
+## Testing Locally
+
+Disaster Recovery can be tested with the local development setup. This may help to gain confidence before performing the procedure on production clusters.
+
+### Preparation
+
+For testing purposes create a local kind-cluster and deploy a Garden cluster into it by following the [Local Development Setup Guide](../deployment/getting_started_locally.md). The recommended way is to use the `gardener-operator` and its respective `make` targets. At the time of writing this, the following commands can be used:
+
+```console
+make kind-multi-zone-up operator-up
+make operator-seed-up
+```
+
+Once everything is up and running, create a couple of resources (`project`, `secret`, `serviceaccount`, ...) used to validate and restore procedure.
+
+Depending on the testcases which should be covered, trigger a credentials rotation, confiugre a HA setup or add additional resources to be encrypted.
+
+Finally, create a service account token and store it away. The token can be used later to validate that existing credentials are still valid after the restore.
```

Comment:
"store it away" sounds weird to me 😄 Also: Where?

Reply:
addressed with fba5051ae93c0363a29a11c343205fd5dfd8a5b1